config/config.go

```diff
@@ -285,8 +285,10 @@ func (c *Config) Validate() error {
 		}
 
 		interp := false
-		fn := func(ast.Node) (interface{}, error) {
-			interp = true
+		fn := func(n ast.Node) (interface{}, error) {
+			if _, ok := n.(*ast.LiteralNode); !ok {
```

Comment:
Comment above here please explaining what this is looking for. :) Its obvious for us since we know the HIL AST.